docs/tutorial/force_calibration.rst

```diff
@@ -464,3 +464,102 @@ When working with small beads or at high laser powers, it is important to verify
+
+    fit = lk.fit_power_spectrum(ps_blocked, model, bias_correction=False, loss_function="lorentzian")
+
+Now plot the robust fit::
+
+    plt.figure()
+    fit.plot()
+    plt.title(
+        f'Robust fit: $f_c$ = {fit.results["fc"].value:.1f}, $D$ = {fit.results["D"].value:.4f}, $f_d$ = {fit.results["f_diode"].value:.1f}'
+    )
+    plt.show()
+
+.. image:: figures/force_calibration/power_spectrum_noise_peak_robust.png
+
+Notice how the model now follows the power spectrum nearly perfectly. The value for `f_diode` has increased significantly, now that it is not abused to reduce the error induced by the outlier. 
+
+This example shows that a robust fitting method is less likely to choke on outliers in the power spectrum data. It is therefore a fair question why one would not use it all the time? First of all, robust fitting leads to a small bias in the fit results for which pylake has no correction. Least-squares fitting also leads to a bias, but this can be corrected with `bias_correction=True`. Secondly, for least-squares fitting, a large pool of literature is available to estimate the errors in the fit of the free parameters, which are implemented in the least-squares fitting routines that Pylake uses. These error estimates are not implemented for robust fitting, and as such, the fit results will show `nan` for the error estimates after a robust fit. However, as will be shown below, the robust fitting results may be used as a start to identify outliers automatically, in order to exclude these from a second, regular least-squares, fit. 
+
+Automated spurious peak detection
+---------------------------------
+
+We will continue the tutorial with the results of the previous section. If you did not yet do that part of the tutorial, please go back and execute the code examples in that section. 
+
+We still have the power spectrum `ps` that was created without blocking or windowing. Here we will use it to identify the peak and automatically obtain frequency exclusion ranges. We will use the method `identify_peaks()` of the `PowerSpectrum` object in order to do so. This method takes a function that accurately models the power spectrum as a function of frequeny, in order to normalize it. It then identifies peaks based on the likelihood of encountering a peak of a certain magnitude in the resulting data set. If we have a "good fit", then the easiest way to get that function is to distill it from our fit parameters and the model that was used during fitting::
+
+    params = [v.value for k, v in fit.results.items() if k in ['fc', 'D', 'f_diode', 'alpha']]
+    model_fun = lambda f: fit.model(f, *params)
```

Comment:
For now this is fine, but it really makes me wonder whether we should have an API to return the solution parameters directly on the result object, since there is a risk these parameters change, or someone wants to fit a model without diode, or a different diode model in the future. It'd also be much more convenient.

Reply:
Yeah I agree. This feels kind of 'hacky'. I'd say it would be nice to have a fit result into which you can immediately plug a frequency, and get a power density. TBC